packages/main/src/Tab.ts

```diff
@@ -61,7 +61,17 @@ const DESIGN_DESCRIPTIONS = {
  * @implements sap.ui.webc.main.ITab
  * @public
  */
-@customElement("ui5-tab")
+@customElement({
+	tag: "ui5-tab",
+	renderer: litRender,
+	template: TabTemplate,
+	styles: css,
+	dependencies: [
+		Icon,
+		Button,
+		CustomListItem,
+	],
+})
 @languageAware
```

Comment:
move this @languageAware decorator to customElement's one?

Reply:
yes, missed that one